Cargo.toml

```diff
@@ -223,6 +223,7 @@ sql = { path = "src/sql" }
 store-api = { path = "src/store-api" }
 substrait = { path = "src/common/substrait" }
 table = { path = "src/table" }
+tests-integration = { path = "tests-integration" }
```

Comment:
Some non-test code need `Database`, so I put this as dependency in several places, any better ideas?

Reply:
That's alright for this PR. Just leave a todo here and we can come back to it later.